Compile/src/Producers/JetSorter.cc

```diff
@@ -7,12 +7,20 @@ void JetSorter::Produce(ZJetEvent const& event,
                         ZJetSettings const& settings) const
 {
     // Iterate over all jet correction levels
+    LOG(DEBUG) << "\n[JetSorter]";
     for (std::map<std::string, std::vector<std::shared_ptr<KJet>>>::const_iterator itlevel =
              product.m_correctedZJets.begin();
-         itlevel != product.m_correctedZJets.end(); ++itlevel) {
+        itlevel != product.m_correctedZJets.end(); ++itlevel) {
+	if(settings.GetDebugVerbosity() > 1) {
+            LOG(DEBUG) << "corr lvl: " << itlevel->first;
+            LOG(DEBUG) << "size before sort: " << product.m_correctedZJets[itlevel->first].size();
```

Comment:
This size check is only here because of the missing Jet investigation right? I think this can be removed as std::sort should work fine.

Reply:
Done.